app/services/analysis/gwas.rb

```diff
@@ -51,7 +71,7 @@ def store_results
     end
 
     def selected_traits
-      @selected_traits ||= analysis.args["phenos"]
+      @selected_traits ||= analysis.meta["phenos"]
```

Comment:
Maybe that is not the best name, but I changed it because at first I intended to keep command line args for GWASSER.R executable. Right now it holds also PT id and in the future it can possibly hold other analysis parameters.